app/src/main/java/edu/calvin/cs262/project01/Messaging.java

```diff
@@ -30,6 +30,12 @@ protected void onCreate(Bundle saveInstanceState) {
         editText = (EditText) findViewById(R.id.editText);
```

Comment:
Add a comment above the class declaration around line 11 about what the Messaging page does.